guide/14-deep-learning/inference-only-text-models-in-arcgis-learn.ipynb

```diff
@@ -0,0 +1,987 @@
+{
```

Comment:
<p>The <s>pretrained</s> <s>transformer model which gets loaded when we instantiate the class object of</s> <strong>ZeroShotClassifier</strong>  model <s>is</s> has been fine-tuned on <a href="https://cims.nyu.edu/~sbowman/xnli/" target="_blank">XNLI</a> corpus which includes 15 languages: Arabic, Bulgarian, Chinese, English, French, German, Greek, Hindi, Russian, Spanish, Swahili, Thai, Turkish, Urdu, and Vietnamese. So this model can be used to classify <strong>multi-lingual</strong> text as well.  </p><p>Below example shows how <s>the classifie</s>r this model can be used to classify an input sequence written in Spanish language. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Made the necessary changes</p>